packages/scripts/internationalization/generators/npm/readme.generator.js

```diff
@@ -0,0 +1,60 @@
+const fs = require('fs');
+const path = require('path');
+const template = require('lodash.template');
+
+const { printInfo, printRunning, printSuccess } = require('../../common/log');
+
+const readmeTemplate = template(`
+# <%= project %> locales
+
+This module contains i18n files for <%= project %> project.
+
+## How to install
+\`\`\`
+yarn install @talend/locales-<%= normalizedName %>
+\`\`\`
+
+## How to use with i18next
+\`\`\`javascript
+// i18n.js
+import i18n from 'i18next';
+
+import merge from 'lodash.merge';
+
+import {
+	namespaces as <%= minimalName %>Namespaces,
+	locales as <%= minimalName %>Locales,
+} from '@talend/locales-<%= normalizedName %>';
+
+i18n
+	.init({
+		ns: [
+			...<%= minimalName %>Namespaces,
+			...myProjectNamespaces
+		],
+		resources: merge(<%= minimalName %>Locales, myProjectLocales),
+	});
+
+export default i18n;
```

Comment:
Please add in the readme where the JSON are located, so if I want to merge assets to use fetch instead of require all langues I can do it.